### PR TITLE
Update diagram tooling to include Visio or PowerPoint

### LINKED ISSUE #921: Update diagram tooling to include Visio or PowerPoint
**Is your feature request related to a problem? Please describe.**

In our last engagement, we used the recommended drawio tooling. We are in the release phase and we have to recreate some of our architecture diagrams to publish in the Azure Architecture Center.

People in the team also think we should not dismiss Microsoft tooling that are at least as good as drawio.

**Describe the solution you'd like**

`docs/design/design-reviews/recipes/preferred-diagram-tooling.md` should include Visio or PowerPoint at least at the same level as drawio.

**Describe alternatives you've considered**

- Choose Visio only. Drawbacks include: no desktop app on the Mac (even though the Web App works great), some customers may not have a license (it's not part of the Office 365 license), there may be legacy diagram habits with drawio
- Choose PowerPoint only. Drawbacks include: the main purpose is not diagramming, Azure icons need to be downloaded separately, there may be legacy diagram habits with drawio
- Choose Visio or Powerpoint only. Drawbacks include: there may be legacy diagram habits with drawio, *.drawio.png and integration with VS code are great features that are unique to drawio.

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by benjguin**
# Pull Request Template

## What are you trying to address

- Update diagram tooling to include Visio or PowerPoint - fixes #921 
- When .drawio format is used, recommend .drawio.png, to complement what was started with #848 

## Description of new changes

- Add Visio and Powerpoint option
- Restructure the page so that the three options are described (Visio, PowerPoint, Drawio)

## For all pull requests

- [x] Link to the issue you are solving (so it gets closed)
- [x] Label the pull request with the appropriate area(s)
- [x] Assign potential reviewers (you may also want to contact them on Teams to ensure timely reviews)
- [ ] Assign the project - Engineering Playbook Backlog
- [x] Assign the pull request to yourself

### CODE REVIEW

File: docs/design/design-reviews/recipes/preferred-diagram-tooling.md
Comment (TessFerrandez) [inline docs/design/design-reviews/recipes/preferred-diagram-tooling.md:14]: I think we should hold off on giving a tool of choice, they all have their uses, so I would tone this down a bit, but we can still mention that it is the preferred format for the Azure Architecture Center if they have explicitly expressed that.

**Review by TessFerrandez — APPROVED** [review]
Thank you,

Approving with a suggestion

File: docs/design/design-reviews/recipes/preferred-diagram-tooling.md
Comment (benjguin) [reply docs/design/design-reviews/recipes/preferred-diagram-tooling.md]: Thanks for the heads up @TessFerrandez. Done.

File: docs/design/design-reviews/recipes/preferred-diagram-tooling.md
Comment (shiranr) [inline docs/design/design-reviews/recipes/preferred-diagram-tooling.md:6]: NIT: perhaps add links to Visio and Power point?

**Review by shiranr — APPROVED** [review]
LGTM, added a small NIT comment :)

File: docs/design/design-reviews/recipes/preferred-diagram-tooling.md
Comment (benjguin) [reply docs/design/design-reviews/recipes/preferred-diagram-tooling.md:6]: Thanks Shiran. 

I hesitated. Then I thought readers of this content would know where to find Visio or PowerPoint, while they might discover drawio tools.

This does not seem to be a strong requirement on your side either. I'll resolve the conversation as is.

PS: The links that would last quite long would probably have been <https://microsoft.com/visio> and <https://microsoft.com/powerpoint>.

**Pull request merged**
